includes/admin_import.php

```diff
@@ -156,127 +159,21 @@ function tsml_import_page()
             }
         }
 
-        // add data source
```

Comment:
This block responsible for polling a new or existing data source for meeting updates and adding them to the buffer now in `tsml_import_data_source($data_source_url, $data_source_name = '', $data_source_parent_region_id = 0, $data_source_change_detect = 'disabled')`